libs/designsystem/src/lib/components/list/list.component.spec.ts

```diff
@@ -2,39 +2,58 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import * as ionic from '@ionic/angular';
 import { createComponentFactory, Spectator } from '@ngneat/spectator';
 import { MockComponent } from 'ng-mocks';
+import { ExpandOperator } from 'rxjs/internal/operators/expand';
```

Comment:
I get a `'ExpandOperator' is declared but its value is never read` linting warning - consider removing it